### ISSUE TITLE
Allow Angled Panels to Open in Sequence

### ISSUE THREAD

**Issue by Libbybapa** [BEEmod/BEE2-items#3015]
It would be a really neat effect if you could connect angled panels (and 1/2 panels) together in a sequence.  When the first panel is activated, as soon as the animation is complete the next panel activates, and so on until they are all open.

**Comment by Libbybapa** [comment]
To be clear, I understand that you can use timers but the delay is larger than I'd like.  I know you can also use other methods for sequential connections, but they are heavy on components and usually make noise.

**Comment by Luke18033** [comment]
I suggested an option for more precise delayers at one point, though panel chaining would probably still be a good thing to have on its own.

**Comment by Danimob** [comment]
You could potentially do this by giving input objects outputs.  Things like the angled panel, flip panel, etc

**Comment by Libbybapa** [comment]
Indeed, that is what I am suggesting, although it would be good if the input was delayed by the amount of time it takes to run the animation of the panel opening.

**Comment by Danimob** [comment]
Great minds think alike
